baseservice/iodevices/file_system/file_system_device_manager_base.py

```diff
@@ -83,9 +85,3 @@ def connect(self):
             create_dir_if_not_exists(self._bookkeeping_folder)
```

Comment:
I think you already have a os.makedirs(exist_ok=True) to do that

Reply:
true. but I still want the chmod... is this really that bad?

Reply:
You could at least change the implementation of the function. You can also imply in its name. (Maybe "ensure_available_directory"?)